packages/react-app/src/components/Account.tsx

```diff
@@ -206,18 +214,18 @@ export default function Account({ minimized }) {
       displayToast()
       return
     }
-    window.ethereum.on('chainChanged', async chainId => {
-      const accounts = await window.ethereum.request({
+    (window.ethereum as any).on('chainChanged', async (chainId: string | number | any) => {
```

Comment:
see https://docs.metamask.io/guide/ethereum-provider.html#chainchanged
this is a string

Reply:
Thanks for pointing this out. I definitely wasn't sure of the type. I will ammend this everywhere.